index.js

```diff
@@ -271,7 +271,31 @@ class CerberusClient {
    * @private
    */
   _executeRequest (requestConfig) {
-    return request(requestConfig)
+    let resp
+    let i
+    for (i = 0; i < DEFAULT_RETRY_ATTEMPT_NUMBER; i++) {
+      resp = request(requestConfig)
+      if (!resp) {
+        return resp
+      } else if (!resp.status_code >= 500) {
```

Comment:
This is a nit, but this line might be more readable, if the if statement was (status_code < 500)